src/webgpu/api/operation/command_buffer/programmable/state_tracking.spec.ts

```diff
@@ -265,10 +338,13 @@ g.test('compatible_pipelines')
   )
   .fn(t => {
     const { encoderType } = t.params;
-    const pipelineA = t.createBindingStatePipeline(encoderType, { a: 0, b: 1, out: 2 });
+    t.skipIfNeedsStorageBuffersInFragmentStageAndHaveNone('storage', encoderType);
+
+    const pipelineA = t.createBindingStatePipeline(encoderType, { a: 0, b: 1, out: 2 }, 'storage');
```

Comment:
nit: (although it's existing code) prefer having explicit 'a.value - b.value' passed in as algorithm, since it's needed context for the `t.expectGPUBufferValuesEqual(outA, new Int32Array([1]));`)

Reply:
done